Apps/Admin/Client/Pages/DashboardPage.razor

```diff
@@ -98,27 +98,59 @@ else
 {
     <MudSkeleton Width="20%" />
     <MudGrid>
-        <MudItem xs="6" lg="6">
+        <MudItem xs="12" lg="4">
             <MudPaper Class="my-2 pa-4" Outlined="true">
                 <MudSkeleton />
                 <MudSkeleton data-testid="skeleton-user-count" />
             </MudPaper>
         </MudItem>
+        <MudItem xs="12" lg="4">
+            <MudPaper Class="my-2 pa-4" Outlined="true">
+                <MudSkeleton />
+                <MudSkeleton data-testid="skeleton-mobile-count" />
+            </MudPaper>
+        </MudItem>
+        <MudItem xs="12" lg="4">
+            <MudPaper Class="my-2 pa-4" Outlined="true">
+                <MudSkeleton />
+                <MudSkeleton data-testid="skeleton-web-count" />
+            </MudPaper>
+        </MudItem>
     </MudGrid>
 }
 else
 {
     <MudText Class="my-3" Typo="Typo.subtitle1">
-        <strong>Recurring Users</strong>
+        <strong>User Counts</strong>
     </MudText>
     <MudGrid>
-        <MudItem xs="6" lg="6">
+        <MudItem xs="12" lg="4">
             <MudPaper Class="my-2 pa-4" Outlined="true">
                 <MudText Align="Align.Center" Typo="Typo.subtitle1">
-                    <strong>User Count</strong>
+                    <strong>Recurring Users</strong>
                 </MudText>
                 <MudText Align="Align.Center" Typo="Typo.subtitle1" data-testid="total-unique-users">
-                    <strong>@TotalRecurringUsers</strong>
+                    <strong>@RecurringUserCountsResult["RecurrentUserCount"]</strong>
+                </MudText>
+            </MudPaper>
+        </MudItem>
+        <MudItem xs="12" lg="4">
+            <MudPaper Class="my-2 pa-4" Outlined="true">
+                <MudText Align="Align.Center" Typo="Typo.subtitle1">
+                    <strong>Mobile Logins</strong>
+                </MudText>
+                <MudText Align="Align.Center" Typo="Typo.subtitle1" data-testid="total-mobile-users">
+                    <strong>@RecurringUserCountsResult["Mobile"]</strong>
```

Comment:
This sort of access can be dangerous if the key doesn't exist (e.g. if the call to the back-end errors out).  I'd suggest creating properties in the code-behind to access these safely.

Reply:
Fixed.